docs/running-locust-distributed.rst

```diff
@@ -85,3 +85,10 @@ Running Locust distributed without the web UI
 =============================================
 
 See :ref:`running-locust-distributed-without-web-ui`
+
+
+Increase Locust's performance
```

Comment:
could also link boomer here, I suspect that has higher overall RPS throughput than any python code ever will